tests/testing/test_dispatch.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import absolute_import
+
+import pytest
+
+from tchannel import TChannel, Request
+from tchannel.errors import BadRequestError
+from tchannel.testing.dispatch import handle
```

Comment:
because if we call handler directly, the return value are not guaranteed. It could be a string or a float to treat as response body while it also could be a response object. This method will convert all possibilities into one type of result which is tchannel response. As client, it would like to check the response, service own can double check their return value will be converted into right response object.